Sources/Discourse/Command.swift

```diff
@@ -33,7 +33,7 @@ public extension Command {
     func populateArguments(from passedArguments: [String]) throws {
         var argumentPairs: [String: String] = [:]
         stride(from: 0, to: passedArguments.count - 1, by: 2).forEach {
-            argumentPairs[passedArguments[$0]] = passedArguments[$0 + 1].escapedCommandLineString
+            argumentPairs[passedArguments[$0].escapedCommandLineString] = passedArguments[$0 + 1].escapedCommandLineString
```

Comment:
I suspect you don't need to escape the value here, but I've left it in in case I'm wrong on that.

Reply:
No, escaping it here makes sense!